mnskshyp

```diff
@@ -0,0 +1,30 @@
+# 100DaysOfCode
+
+MANAS KASHYAP
+
+2nd year Student MNIT Jaipur
+
+**Subtask #1: Improving DSA**
+
+-   C++ Theory
+-   C++ STL
+-   DSA Concepts
+-   Practising on geeksforgeeks
+
+
+**Subtask #2: Competitive Coding**
+
+Codechef: (Long Challenge, Cook-off, Lunchtime)
+
+-   2 star coder
+
+Hackerrank:
+
+-   Problem Solving
+
+**Subtask #3: Web-Development**
+
+-   Frontend
+    -   HTML
+    -  CSS
```

Comment:
I will merge when the requested changes are done and progress is added for some day